File: Python/md5test.py
from random import randint
import sys, time, hashlib

startTime = time.time()
c=0
m=10000000
while(c<m):

  n0=randint(0,9999999999)
  n1=randint(0,9999999999)
  nn=hashlib.md5((str(n0) + str(n1)).encode('utf-8')).hexdigest()
  c=c+1

print ('Python: Version ' + sys.version)
print ('Execution time (seconds): {0}'.format(time.time() - startTime))

File: Ruby/md5test.rb
require 'digest'

start_time = Time.now
c = 0
m = 10000000

while c < m
  n0 = rand(9999999999)
  n1 = rand(9999999999)
  nn = Digest::MD5.hexdigest(n0.to_s << n1.to_s)
  c += 1
end

puts "Ruby version: #{RUBY_VERSION}"
puts "Execution time (seconds): #{Time.now - start_time}"
